src/PointwiseFunctions/GeneralRelativity/Ricci.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Defines functions to calculate the Ricci tensor
+
+#pragma once
+
+#include "DataStructures/Tensor/IndexType.hpp"
+#include "DataStructures/Tensor/Symmetry.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes Ricci tensor from the (spatial or spacetime)
+ * Christoffel symbol of the second kind and its derivative.
+ *
+ * \details Computes Ricci tensor \f$R_{ab}\f$ as:
+ * \f$ R_{ab} = 1/2 ( \partial_c \Gamma^{c}_{ab} - \partial_{(b}
+ * \Gamma^{c}_{a)c}
+ * + \Gamma^{d}_{ab}\Gamma^{c}_{cd} - \Gamma^{d}_{ac} \Gamma^{c}_{bd}  ) \f$
+ * where \f$\Gamma^{a}_{bc}\f$ is the Christoffel symbol of the second kind.
+ */
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
```

Comment:
Please put in namespace gr.

Reply:
Christoffel and Ricci are now in namespace gr